keyboards/ropro/ropro.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
The second portion of this definition needs to be a multi-dimensional array (an array of arrays).

Lines 5-19 should be:

```c
#define LAYOUT( \
         K10, K20, K30, K40, K50, K60, K70, K80, K90, K100, K110, K120, \
         K11, K21, K31, K41, K51, K61, K71, K81, K91, K101, K111, K121, \
    K02, K12, K22, K32, K42, K52, K62, K72, K82, K92, K102, K112, K122, \
    K03, K13, K23, K33, K43, K53, K63, K73, K83, K93, K103, K113, K123, \
    K04, K14, K24, K34, K44, K54, K64, K74, K84, K94, K104, K114, K124, \
    K05, K15, K25, K35, K45, K55, K65, K75, K85, K95, K105, K115, K125 \
) { \
    { KC_NO, K10, K20, K30, K40, K50, K60, K70, K80, K90, K100, K110, K120 }, \
    { KC_NO, K11, K21, K31, K41, K51, K61, K71, K81, K91, K101, K111, K121 }, \
    { K02,   K12, K22, K32, K42, K52, K62, K72, K82, K92, K102, K112, K122 }, \
    { K03,   K13, K23, K33, K43, K53, K63, K73, K83, K93, K103, K113, K123 }, \
    { K04,   K14, K24, K34, K44, K54, K64, K74, K84, K94, K104, K114, K124 }, \
    { K05,   K15, K25, K35, K45, K55, K65, K75, K85, K95, K105, K115, K125 } \
}
```

Reply:
Argh, I didn't even notice that... that'd be why Travis is sad 😆

Reply:
Also, there should be no spaces after the backslash.  It is what the compiler error you saw above was from.